### PR TITLE
Add/curation model

### PR DESCRIPTION
**Pull request by dglemos**
Replaces #8 

Add endpoints:
- `gene2phenotype/api/add/curation` add a new curation record (only supports "raw data")
- `gene2phenotype/api/curations` lists all the entries under curation
- `gene2phenotype/api/curation/<str:stable_id>` displays the curation entry info for a specific G2P stable ID

These endpoints are only available for authenticated users.

### CODE REVIEW

File: gene2phenotype_project/gene2phenotype_app/views.py [hunk 1]
```diff
@@ -326,6 +328,10 @@ def list(self, request, *args, **kwargs):
         serializer = LocusGenotypeDiseaseSerializer(queryset)
         return Response(serializer.data)
 
+
+"""
+
+"""
```
Comment (seeta-ramaraju) [inline gene2phenotype_project/gene2phenotype_app/views.py:334]: if these are not required, remove them

File: gene2phenotype_project/gene2phenotype_app/urls.py [hunk 2]
```diff
@@ -26,6 +26,11 @@ def perform_create(self, serializer):
     path('gene2phenotype/api/add/phenotype/', views.AddPhenotype.as_view(), name="add_phenotype"),
     path('gene2phenotype/api/add/publication/', views.AddPublication.as_view(), name="add_publication"),
     path('gene2phenotype/api/lgd/<str:stable_id>/add_panel', views.LocusGenotypeDiseaseAddPanel.as_view(), name="lgd_add_panel"),
+
+    ### Curation endpoints ###
+    path('gene2phenotype/api/add/curation/', views.AddCurationData.as_view(), name="add_curation_data"),
+    path('gene2phenotype/api/curation/list', views.ListCurationEntries.as_view(), name="list_curation_entries"),
```
Comment (seeta-ramaraju) [inline gene2phenotype_project/gene2phenotype_app/urls.py:32]:
```suggestion
    path('gene2phenotype/api/curations', views.ListCurationEntries.as_view(), name="list_curation_entries"),
```

Comment (dglemos) [reply gene2phenotype_project/gene2phenotype_app/urls.py]: Updated

**Review by olaaustine — APPROVED** [review]
Thank you @dglemos. 
Looks good to me

**Pull request merged**